pkg/buildah/build.go

```diff
@@ -98,6 +102,39 @@ func buildCmd(c *cobra.Command, inputArgs []string, sopts saveOptions, iopts bui
 		defer iopts.Logwriter.Close()
 	}
 
+	if len(iopts.File) == 0 {
+		ctx, err := getContextDir(inputArgs)
+		if err != nil {
+			return err
+		}
+		isStat := func(ctxDir, file string) (string, error) {
+			ctrfile := filepath.Join(ctxDir, file)
+			fileInfo, err := os.Stat(ctrfile)
+			var foundCtrFile string
+			if err != nil {
+				return "", fmt.Errorf("cannot find Containerfile or Dockerfile in context directory: %w", err)
+			}
+			// The file exists, now verify the correct mode
+			if mode := fileInfo.Mode(); mode.IsRegular() {
+				foundCtrFile = ctrfile
+			} else {
+				return "", fmt.Errorf("assumed %s %q is not a file", file, ctrfile)
+			}
+			return foundCtrFile, nil
+		}
+
+		defaultFileNames := []string{"Dockerfile", "Containerfile", "Kubefile", "Sealfile"}
```

Comment:
Only use Sealosfile in the feature.

Reply:
> Only use Sealosfile in the feature.

Do we need to be compatible with the previous logic?